servicex/query.py

```diff
@@ -63,6 +63,10 @@
 logger = logging.getLogger(__name__)
 
 
+class ServiceXException(Exception):
+    """ Something happened while trying to carry out a ServiceX request """
+
```

Comment:
Didn't this already exist - in `utils.py` - which I can't find now... Perhaps that got lost in the branch.

Reply:
Yup, it got lost in the branch ... I can maybe do a second PR to restore parts of it?